packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -49,6 +51,11 @@ const propTypes = {
    * Override that shows a known "No Data" display. Used when there is known to be no value for a given clinical result concept at a specific datetime.
    */
   hasResultNoData: PropTypes.bool,
+  /**
+   * @private
+   * The intl object to be injected for translations.
+   */
+  intl: PropTypes.shape({ formatMessage: PropTypes.func }),
```

Comment:
I believe this will be displayed on the props table, have we checked with UX/Scott if the wordings and the name are fine?

Reply:
Let me know if the idea is to tackle all the doc-related changes in https://jira2.cerner.com/browse/UXPLATFORM-8568

Reply:
Clinical Result Blood Pressure already uses the intl prop: https://github.com/cerner/terra-clinical/blob/main/packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx#L56 but it doesn't show up in the props table: https://engineering.cerner.com/terra-ui/components/terra-clinical-result/clinical-result/clinical-result-blood-pressure